### ISSUE TITLE
NextJsSite: Only one deploy works, then errors on EdgeLambdaVersion custom resource

### ISSUE THREAD

**Issue by dkershner6** [serverless-stack/serverless-stack#876]
```
prod-DKershnercom-DKershnercomWebAppStack | CREATE_FAILED | Custom::SSTEdgeLambdaVersion | SiteMainEdgeLambdaVersionE9F7ABA9c4257fb741b847639a07c1f8579c1cc9 Received response status [FAILED] from custom resource. Message returned: Alias already exists: arn:aws:lambda:us-east-1:***:function:prod-dkershnercom-dkershnercomwebappstack-m-jrjyielhfijtyovzknab:live
```

Note that I am deploying into us-west-2 (not us-east-1), in case that is a clue.

The full log stream from the CustomResource function is attached (only my aws account id was redacted).

[log-events-viewer-result.csv](https://github.com/serverless-stack/serverless-stack/files/7272060/log-events-viewer-result.csv)

**Comment by fwang** [comment]
Hi @dkershner6, I'm taking a looking into it!

**Comment by fwang** [comment]
@dkershner6 this is fixed in v0.45.0. Give it a try and let me know if it works now.

**Comment by dkershner6** [comment]
Tried 5 deploys, all worked swimmingly.

Thanks for jumping on this so quickly, @fwang .
